.vscode/settings.json

```diff
@@ -10,5 +10,7 @@
       "source.organizeImports": true
     }
   },
-  "python.linting.pylintEnabled": false
-}
+  "python.linting.pylintEnabled": false,
+  "python.linting.flake8Enabled": false,
+  "python.linting.enabled": false
```

Comment:
Why are we disabling pylint when we're making our code compatible? Not just pylint, but we have disabled the default lint as well. I would advice against it.

Reply:
Linting is disabled for me I accidentally commited that file as well :joy: